{
    "topic": "Gov Commands Errors",
    "pageNumber": "1",
    "type": "Gov Error - Pull Requests Not Processed",
    "definition": {
        "text": "There was a problem trying to process the pull requests at the Superalgos repository",
        "updated": 1611129378492,
        "translations": [
            {
                "language": "DE",
                "text": "Es gab ein Problem bei der Bearbeitung der Pull Requests im Superalgos Repository",
                "updated": 1638746677069
            },
            {
                "language": "RU",
                "text": "Возникла проблема при попытке обработать запросы на извлечение (\"pull request\") в репозитории Superalgos",
                "updated": 1645627422470
            }
        ]
    },
    "paragraphs": [
        {
            "style": "Text",
            "text": "Check the following section for details on what may have happened. After you fix the issue that prevented the Gov PRs Command to run, you may try again.",
            "updated": 1611152997869,
            "translations": [
                {
                    "language": "DE",
                    "text": "Lesen Sie den folgenden Abschnitt, um zu erfahren, was passiert sein könnte. Nachdem Sie das Problem behoben haben, das die Ausführung des Befehls Gov PRs verhindert hat, können Sie es erneut versuchen.",
                    "updated": 1638746717268
                },
                {
                    "language": "RU",
                    "text": "Подробную информацию о том, что могло произойти, см. в следующем разделе. После устранения проблемы, которая не позволила запустить Gov PRs Command, вы можете повторить попытку.",
                    "updated": 1645627452165
                }
            ]
        },
        {
            "style": "Title",
            "text": "Error Info",
            "updated": 1611154606702,
            "translations": [
                {
                    "language": "RU",
                    "text": "Информация об ошибке",
                    "updated": 1645627461013
                }
            ]
        },
        {
            "style": "Text",
            "text": "In the following section, you will find the information recorded when the error occurred.",
            "translations": [
                {
                    "language": "DE",
                    "text": "Im folgenden Abschnitt finden Sie die Informationen, die beim Auftreten des Fehlers aufgezeichnet wurden.",
                    "updated": 1638746731702
                },
                {
                    "language": "RU",
                    "text": "В следующем разделе вы найдете информацию, записанную при возникновении ошибки.",
                    "updated": 1645627467846
                }
            ]
        },
        {
            "style": "Note",
            "text": "You will only see the information if you opened this page as a result of the error actually occurring.",
            "updated": 1611153032408,
            "translations": [
                {
                    "language": "DE",
                    "text": "Die Informationen werden nur angezeigt, wenn Sie diese Seite geöffnet haben, weil der Fehler tatsächlich aufgetreten ist.",
                    "updated": 1638746793412
                },
                {
                    "language": "RU",
                    "text": "Вы увидите информацию только в том случае, если вы открыли эту страницу в результате того, что ошибка действительно произошла.",
                    "updated": 1645627475394
                }
            ]
        },
        {
            "style": "Subtitle",
            "text": "Error Message",
            "translations": [
                {
                    "language": "DE",
                    "text": "Error Nachricht",
                    "updated": 1638746754945
                }
            ]
        },
        {
            "style": "Text",
            "text": "The following is the Error Message of the Exception that occurred at the Client:",
            "translations": [
                {
                    "language": "DE",
                    "text": "Im Folgenden wird die Fehlermeldung der beim Client aufgetretenen Ausnahme angezeigt:",
                    "updated": 1638746808346
                },
                {
                    "language": "RU",
                    "text": "Ниже приведено сообщение об ошибке исключения, произошедшего на клиенте:",
                    "updated": 1645627491359
                }
            ]
        },
        {
            "style": "Placeholder",
            "text": "errorMessage"
        },
        {
            "style": "Subtitle",
            "text": "Error Stack"
        },
        {
            "style": "Text",
            "text": "The following is the Error Stack of the Exception that occurred at the Client:",
            "translations": [
                {
                    "language": "DE",
                    "text": "Im Folgenden ist der Error Stack der beim Client aufgetretenen Ausnahme dargestellt:",
                    "updated": 1638746830116
                },
                {
                    "language": "RU",
                    "text": "Ниже приведен стек ошибок исключения, произошедшего на клиенте:",
                    "updated": 1645627498475
                }
            ]
        },
        {
            "style": "Placeholder",
            "text": "errorStack"
        },
        {
            "style": "Subtitle",
            "text": "Error Code"
        },
        {
            "style": "Text",
            "text": "The following is the Error Code of the Exception that occurred at the Client:",
            "translations": [
                {
                    "language": "DE",
                    "text": "Im Folgenden wird der Fehlercode der beim Client aufgetretenen Ausnahme angegeben:",
                    "updated": 1638746868393
                },
                {
                    "language": "RU",
                    "text": "Ниже приведен код ошибки исключения, произошедшего у клиента:",
                    "updated": 1645627505034
                }
            ]
        },
        {
            "style": "Placeholder",
            "text": "errorCode"
        },
        {
            "style": "Subtitle",
            "text": "Error Details",
            "updated": 1611154543778
        },
        {
            "style": "Text",
            "text": "The following are all the Error details of the Exception that occurred at the Client:",
            "updated": 1611154675224,
            "translations": [
                {
                    "language": "DE",
                    "text": "Im Folgenden werden alle Fehlerdetails der beim Client aufgetretenen Ausnahme aufgeführt:",
                    "updated": 1638746879966
                },
                {
                    "language": "RU",
                    "text": "Ниже приведены все детали ошибки исключения, произошедшего у клиента:",
                    "updated": 1645627521600
                }
            ]
        },
        {
            "style": "Placeholder",
            "text": "errorDetails",
            "updated": 1611154555419
        },
        {
            "style": "Text",
            "text": ""
        }
    ]
}